SwiftWeatherTests/UnitTests/ForecastDateTimeSpec.swift

```diff
@@ -23,11 +21,10 @@ class ForecastDateTimeSpec: QuickSpec {
     describe("#shortTime") {
       it("should return the correct shortTime string with format HH:mm") {
         var forecastDateTime = ForecastDateTime(1488096060)
-        expect(forecastDateTime.shortTime).to(equal("19:01"))
+        expect(forecastDateTime.shortTime).to(equal("01:01"))
```

Comment:
Because the different device has a different timezone, we need to add timezone to `ForecastDateTime` like

```
struct ForecastDateTime {
  init(_ date: Double, timeZone: TimeZone) {
    rawDate = date
  }
}
```

Then in the test, we already have a fixed timezone like `let testTimeZone = TimeZone(abbreviation: "UTC+11:00")!` to pass in.

```
var forecastDateTime = ForecastDateTime(1488096060, timeZone: testTimeZone)
```

Reply:
Amazing solution, I will implement it :D